tensorboard/defs/defs.bzl

```diff
@@ -32,11 +32,21 @@ def tf_js_binary(
         deps,
         visibility = None,
         dev_mode_only = False,
+        includes_polymer = False,
         **kwargs):
     """Rules for creating a JavaScript bundle.
 
-    Please refer to https://bazelbuild.github.io/rules_nodejs/Built-ins.html#rollup_bundle
-    for more details.
+    Args:
+        name: Name of the target.
+        compile: whether to compile when bundling.
```

Comment:
Maybe worth noting in these docs that `compile` and `includes_polymer` are only used internally.